### DISCUSSION TITLE
Has anyone tried to use Actix Web for Api and Remix for the Front End?

### DISCUSSION THREAD

**Discussion by goldcoders** [remix-run/remix#1957] [Proposals]
I search google for example but cant find any example,

I wondering , if anyone has working example.

I think using actix web , postgresql and remix + react js 

would be awesome to create in production site

Combine the Fastest rust web framework with Remix 

anyway , I'll try to create one myself 

I wanna see the raw performance , Im looking to use it on my next project :)

**Comment by OldManFroggy** [comment]
yes and Axum, pretty straight forward just use fetch as you would normally.

On performance, most backends will have other bottle necks than pure API
performance in RUST/GO/NodeJS, etc...
